### PR TITLE
Add template variables to add info about who is inviting to work in a…

### PR DESCRIPTION
**Pull request by spikeboom**
…n issue.

## Description

> Description of the pull request

## Changes

- Where your changes apply

## Issue

> Closes #000

## Impacted Area

> Main page

## Steps to test

Steps needed to reproduce the scenario from this change to validate if the pull request solve this issue

- Create / log user
- Create Task
- Other steps

### CODE REVIEW

File: locales/result/br.json [hunk 1]
```diff
@@ -87,7 +87,7 @@
   "mail.assign.remove.message": "<p>Você foi removido da tarefa <a href=\"{{url}}\">{{title}}</a> no Gitpay</p><p>O responsável pela tarefa enviou a seguinte mensagem:</p><p>{{message}}</p>",
   "mail.assign.remove.owner.subject": "O usuário foi desassociado de uma tarefa que você criou no Gitpay",
   "mail.assign.remove.owner.message": "<p>O usuário {{user}} ({{email}}) foi desassociado da tarefa <a href=\"{{url}}\">{{title}}</a> no Gitpay</p><p>A razão foi:</p><p>{{message}}</p>",
-  "mail.invite.send.action": "Você recebeu um convite para trabalhar em uma tarefa do Gitpay :-)",
+  "mail.invite.send.action": "Você recebeu um convite de {{name}} para trabalhar em uma tarefa do Gitpay :-)",
```
Comment (alexanmtz) [inline locales/result/br.json:90]: @spikeboom please do not change here cause this is a generated file with the final translations.

File: locales/result/en.json [hunk 2]
```diff
@@ -87,7 +87,7 @@
   "mail.assign.remove.message": "<p>You were unassigned from an issue <a href=\"{{url}}\">{{title}}</a> on Gitpay</p><p>Reason:</p><p>{{message}}</p>",
   "mail.assign.remove.owner.subject": "The user was unassigned of an issue you created on Gitpay",
   "mail.assign.remove.owner.message": "<p>The user {{user}} ({{email}}) was unassigned of an issue <a href=\"{{url}}\">{{title}}</a> on Gitpay</p><p>The reason was:</p><p>{{message}}</p>",
-  "mail.invite.send.action": "You were invited to solve an issue added to Gitpay :-)",
+  "mail.invite.send.action": "You were invited by {{name}} to solve an issue added to Gitpay :-)",
```
Comment (alexanmtz) [inline locales/result/en.json:90]: @spikeboom same here

File: modules/tasks/taskInvite.js [hunk 3]
```diff
@@ -14,7 +14,7 @@ module.exports = Promise.method(function ({ id }, { message, email }) {
       i18n.setLocale(language)
       SendMail.success(
         { email, language },
-        i18n.__('mail.invite.send.action'),
+        i18n.__('mail.invite.send.action', { name: name }),
         `${i18n.__('mail.invite.send.message', {
           title: task.title,
           url: taskUrl,
```
Comment (alexanmtz) [inline modules/tasks/taskInvite.js:20]: @spikeboom we should add the template on the message body as well

File: locales/result/br.json (hunk 1 above)
Comment (spikeboom) [reply locales/result/br.json]: solved

File: locales/result/en.json (hunk 2 above)
Comment (spikeboom) [reply locales/result/en.json]: solved

File: modules/tasks/taskInvite.js (hunk 3 above)
Comment (spikeboom) [reply modules/tasks/taskInvite.js:21]: solved

**Comment by alexanmtz** [comment]
Hello @spikeboom, why this PR was closed? I just asked for some changes and we could proceed. Let me know if you need any help.

**Pull request closed without merging**